### ISSUE TITLE
Accessing Child/Sub pages from Parent/Top level page 

### ISSUE THREAD

**Issue by mhoelter** [SpinaCMS/Spina#559]
Hello,

Is it possible to access the child/nested pages, specifically their content, from a parent page?

For example, if you have a parent page that lists blog posts, with several child pages being the posts, how would you be able to access the title, content, etc. from the child pages on the parent page?

**Comment by Bramjetten** [comment]
Yes, you can use `Current.page.children` to get the children and just call `content` method on a child page.

You probably want to check out [Resources (SpinaCMS Guides)](https://www.spinacms.com/guide/themes/resources#main) which uses blogposts as an example.
